docs/build/guides/tokens/custom-sac-admin.mdx

```diff
@@ -0,0 +1,161 @@
+---
+title: Set a custom admin account for a Stellar Asset Contract (SAC)
+hide_table_of_contents: true
+description: Set a custom administrator account on a deployed SAC
+---
+
+The [Stellar Asset Contract (SAC)](../../../tokens/stellar-asset-contract.mdx) includes functionality to set a custom administration account on the contract itself. This can be seen as a security precaution, similar to locking an asset's issuer account after creating the total supply of tokens. It should also be noted that an asset's issuer account can have a distinct set of signatures and weights, and these are unrelated to the SAC admin address.
```

Comment:
I don't think this can be seen as security precaution. The issuer account is not locked in this case and it can keep miniting/burning/clawbacking tokens (if enabled). We can say that the issuer account _can_ be locked thus moving the token administration _exclusively_ to Soroban. But I don't think this has anything to do with security precautions. Basically, it's important to note that changing the admin does not automatically revoke any permissions from the original issuer.

Reply:
you're right. i think i was making an unfounded connection between changing the SAC admin and modifying the weights/signers for the issuer `G...` address. I'm cleaning up this language in a new PR